speech_final.py

```diff
@@ -0,0 +1,49 @@
+import pyaudio
+import asyncio
+import sys
+import websockets
+import time
+import json
+import argparse
+import beepy
+
+FORMAT = pyaudio.paInt16
+CHANNELS = 1
+RATE = 16000
+CHUNK = 8000
+
+audio_queue = asyncio.Queue()
+
+def callback(input_data, frame_count, time_info, status_flag):
+    audio_queue.put_nowait(input_data)
+    return (input_data, pyaudio.paContinue)
+
+async def run(key):
+    clock_cursor = 0.
+    audio_cursor = 0.
+    transcript_cursor = 0.
+
+    extra_headers={
+        'Authorization': 'Token {}'.format('key')
+    }
+    try:
+        async with websockets.connect('wss://api.deepgram.com/v1/listen'):
+            print('success')
```

Comment:
is this incomplete? it doesn't seem to do anything

Reply:
OOPS. I screwed this up purposely for the TWIP demo. Lemme fix